src/frontends/onnx/tests/models/model_editor/reference/subgraph__input_edge_from_tensor_with_multiple_consumers_2.prototxt

```diff
@@ -147,17 +147,11 @@ graph {
     }
   }
   output {
-    name: "mul2"
+    name: "add2"
```

Comment:
After the change outputs list on a model matches order of outputs passed by a user.